src/shared/components/community/communities.tsx

```diff
@@ -67,7 +68,7 @@ export class Communities extends Component<any, CommunitiesState> {
     this.handleListingTypeChange = this.handleListingTypeChange.bind(this);
 
     // Only fetch the data if coming from another route
-    if (FirstLoadService.isFirstLoad) {
+    if (!isBrowser() || FirstLoadService.isFirstLoad) {
```

Comment:
If everywhere `isFirstLoad` is used needs to be ORed with `!isBrowser()`, it's easier to move that logic to the `isFirstLoad` function.

Reply:
I was hesitant to change the functionality of `isFirstLoad` in the case it was used for server initialization. If `isFirstLoad` is assumed to be browser first load I'll move the check there.